src/v/config/configuration.cc

```diff
@@ -1840,6 +1840,13 @@ configuration::configuration()
       "the data directory. Redpanda will refuse to start if it is not found.",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       false)
+  , enable_storage_space_manager(
+      *this,
+      "enable_storage_space_manager",
+      "Enable the storage space manager that coordinates and control space "
+      "usage between log data and the cloud storage cache.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      true)
```

Comment:
I don't think the behavior you're implementing warrant doing anything differently in the face of skew, but I'm curious if you've given thought to whether this should be configurable/controllable at the node level. A few scenarios come to mind when space usage may vary across nodes
- when there is some hypothetical retention bug that consumes the space on a a single node but not others
- when there is heterogeneous hardware
- when there is severe partition-size skew

Just thinking whether during an incident it'll be desirable to have a way to flip this off for one but not all nodes.

Reply:
this makes a lot of sense. i'd even assume that we'd want to be able to turn it off on a per node basis without needing a restart. at least i think that we dont' allow that later feature so we'd need an admin endpoint.

Reply:
I've added a ticket to track this here. It's a good idea: https://github.com/redpanda-data/redpanda/issues/11192